projects/tuist/fixtures/app_with_spm_dependencies/Tuist/Dependencies.swift

```diff
@@ -8,16 +10,20 @@ let packages: [Package] = [
     .package(url: "https://github.com/pointfreeco/swift-composable-architecture", .upToNextMinor(from: "0.34.0")),
     .package(url: "https://github.com/iterable/swift-sdk", .upToNextMajor(from: "6.4.0")),
     .package(url: "https://github.com/Trendyol/ios-components", .revision("c9260bfe203a16a278eca5542c98455eece98aa4")),
+    .local(path: "LocalSwiftPackage"),
 ]
 
+let configuration: Configuration = .configuration(options: .options())
```

Comment:
I think the default value of the options should correspond to what you'd get _without_ specifying the options.

Additionally, why are the `options` wrapped inside `.configuration`? Can't the property be `packageGenerationOptions: ["LocalSwiftPackage": .options(disableSynthesizedResourceAccessors: false)]`? I'd find that much easier to read and understand than the current state of things.

Reply:
I think it's wrapped in case we want to add customization of other part of the generated project, but I agree it's simpler without it and I guess we can always add it later if we want, adding a function overload to keep it non breaking

Reply:
> I think the default value of the options should correspond to what you'd get without specifying the options.

The default initializer corresponds to the default values for options defined in `Project.swift`, which are different from the dependencies ones.
At most we could add a `dependencyOptions` method that defaults to the default for dependencies, what do you think?

Reply:
Yea the wrapper was created to allow customization of both `Project.Options` & `Project.ResourceSynthesizers` but I found that `Project.ResourceSynthesizers` support is not straightforward and requires some planning around target dependencies. I do think that we will need to support `Project.ResourceSynthesizers` customization in future as well so that is why I kept `Configuration` but I can remove it if you feel strongly.


Regarding default values: As Daniele mentioned `PackageInfoMapper` had a different default behavior for dependencies than `Project.swift` so we maintained that behavior but I think `dependencyOptions` makes sense rather than in-line definition which is difficult to discover for users.

Reply:
>  I do think that we will need to support Project.ResourceSynthesizers customization in future as well so that is why I kept Configuration but I can remove it if you feel strongly.

We might need to add another property instead then? I'd like to see the proposed API for `ResourceSynthesizers` before making the call whether we want the `Configuration` abstraction or not.